website/src/pages/roadmap.md

```diff
@@ -61,7 +61,7 @@ Next major release : [0.11.0](https://issues.apache.org/jira/projects/HUDI/versi
 |Feature|Target Release|Tracking|
 |------------|--------|-----------|
 |Native support for AWS Glue Metastore   |0.11|[HUDI-2757](https://issues.apache.org/jira/browse/HUDI-2757)|
-|BigQuery and Snowflake external table integration   |0.11|[RFC-34](https://github.com/apache/hudi/pull/4503)|
+|BigQuery and Snowflake external table integration   |0.12|[RFC-34](https://github.com/apache/hudi/pull/4503)|
```

Comment:
we're still making last min effort to make it. if we get it in 0.11, will put it back.